src/main/resources/db_scripts/db_script_v0.sql

```diff
@@ -0,0 +1,45 @@
+create table "workflow" (
+  "name" VARCHAR(45) NOT NULL UNIQUE,
+  "is_active" BOOLEAN NOT NULL,
+  "created" TIMESTAMP NOT NULL,
+  "updated" TIMESTAMP,
```

Comment:
As I previously said, I would set updated to creation time on creation. Se I would make this not null as well
```suggestion
  "updated" TIMESTAMP NOT NULL,
```

Reply:
Yes, I think there is bug for that and it will be fixed in separate PR.

Reply:
Another issue #7